pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -396,6 +397,16 @@ func (os *OpenStack) GetMaxVolLimit() int64 {
 	return defaultMaxVolAttachLimit
 }
 
+// GetFreeCapacity returns free capacity of the block storage
```

Comment:
Would be nice to say that result is expected in bytes. Which is also a bit weird to me.

Reply:
https://docs.openstack.org/api-ref/block-storage/v3/#limits-limits i did not find the role/permission list for csi-cinder. 
I think this feature should be optional.

Reply:
fixed, changed to Gb